cms/djangoapps/contentstore/views/course.py

```diff
@@ -1245,8 +1179,8 @@ def textbooks_list_handler(request, course_key_string):
                     textbook["id"] = tid
                     tids.add(tid)
 
-            if not any(tab['type'] == PDFTextbookTabs.type for tab in course.tabs):
-                course.tabs.append(PDFTextbookTabs())
+            if not any(tab['type'] == 'pdf_textbooks' for tab in course.tabs):
+                course.tabs.append(CourseTab.from_json({'type': 'pdf_textbooks'}))
```

Comment:
Why convert this from json, rather than just being explicit about the type of the PDF tab?